www/squid/src/opnsense/mvc/app/controllers/OPNsense/Proxy/forms/main.xml

```diff
@@ -131,7 +131,7 @@
                 <id>proxy.general.cache.local.enabled</id>
                 <label>Enable local cache</label>
                 <type>checkbox</type>
-                <help>Enable or disable the local cache. Only UFS directory cache type is supported. Do not enable on embedded systems with SD or CF cards as this will wear down your drive.</help>
+                <help>Enable or disable the local cache. Only UFS directory cache type is supported. Do not enable on embedded systems with SD or CF cards as this will wear down your drive. Do not enable with multiple workers!</help>
```

Comment:
We should put that in a constraint instead. More effective than documenting this here loosely.

Reply:
Yes, i also thought about switching the cache_dir automatically to rock store based on the worker choice, but the cache settings for rock store are different from the ufs cache settings. http://www.squid-cache.org/Doc/config/cache_dir/

Reply:
@fichtner Would you mind switching the cache_dir completely to rock? Then it would be compatible with both variants. I would close this and make a new pull?

Reply:
I think we shouldn't aim for breaking changes for a hotfix scope. Constraint and fixing the glob pattern seems better than editing the help text or modifying runtime behaviour.

Reply:
Ok to be honest this may be more tricky than I thought. I would propose the following: switch this hint to the new option, not to the existing option